File: C/Ex35.c
#include<stdio.h>
#include<stdlib.h>

//Função principal do programa
int main(int argc, char const *argv[])
{
// PROGRAMA 35 - ARMAZENAR 20 VALORES EM UM VETOR , ENTRAR COM UMA CTE MULTIPLICATIVA...
//                 ...MULTIPLICAR CADA UM DOS VALORES DO VETOR POR ESSA CTE...
//           ... ARMAZENAR E EXIBIR O RESULTADO NAS RESPECTIVAS POSIÇÕES DO VETOR 


//Declaração das variáveis
int cte, res;

//Exibe na tela
printf("ENTRAR COM 20 VALORES EM UM VETOR, MULTIPLICAR POR UMA CTE E EXIBIR O RESULTADO");

//Declarando e populando o vetor com 20 posições
int numeros[20];

//Looping para popular o vetor
printf("\nDigite os 20 numeros:\n");
for (int i=0; i<20; i++){
    scanf("%i", &numeros[i]);
}
printf("Digite o valor da constante: ");
scanf("%i", &cte);

//Looping para exibir os novos valores do vetor
printf("O vetor multiplicado pela constante e:\n");
for (int i=0; i<20; i++){ //Varrendo o vetor
    res=cte*numeros[i]; //Multiplicando a constante pelo vetor
    printf("%i\n",res);  //Exibe o resultado da multiplicação da constante pelo vetor
}
return 0;
}

File: PYTHON/Ex35.py
''' Vetores - Armazenar vinte valores em um vetor. Após a digitação, entrar com uma cte
multiplicativa que deverá multiplicar cada um dos valores de vetor e armazenar o resultado
no próprio vetor, na respectiva posição...'''

print("Entrar com 20 valores em um vetor, multiplicar por uma cte e exibir o resultado")

numeros = []

for i in range(0,3,1): #range ( parte do 0, vai até inteiro menor que 20, no passo 1 em 1 )  
    num = int(input('Digite um número: '))
    numeros.append(num)

cte = int(input('Digite o valor da constante: '))

print('O vetor multiplicado pela constante é: ')
for i in range (0,3,1): #varrendo o vetor
   res = cte*numeros[i]
   print(res)